frontend/lib/ui/page.tsx

```diff
@@ -41,6 +41,9 @@ function getSiteBaseName(siteType: SiteChoice): string {
 
     case "NORENT":
       return "NoRent.org";
+
+    case "EVICTIONFREE":
+      return "EvictionFree.org";
```

Comment:
We may need to change this based on what our final domain name is, so we should file an issue to address this later.